### PR TITLE
Add missing translations en

### PR DESCRIPTION
**Pull request by paulinebessoles**

### CODE REVIEW

**Comment by Quentinchampenois** [comment]
Hi @paulinebessoles !

Thanks for these added keys ! Can I just ask you to add them too in the `fr.yml` and `nl.yml` file please ?

**Comment by armandfardeau** [comment]
> Hi @paulinebessoles !
> 
> Thanks for these added keys ! Can I just ask you to add them too in the `fr.yml` and `nl.yml` file please ?

Decidim-module-cookies is on crowdin so you must not add keys manually.

**Comment by Quentinchampenois** [comment]
Ok you are right @armandfardeau, sorry @paulinebessoles seems good to me !

File: config/locales/en.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ en:
         consent_modal:
           description: This modal gives the opportunity to filter the cookies you
             want.
+          privacy_policy:
+            name: privacy policy
+            text: To learn more, please read our {privacyPolicy}.
```
Comment (armandfardeau) [inline config/locales/en.yml:14]: I'm not sure about the variable interpolation, can you check it's not 
```suggestion
            text: To learn more, please read our %{privacyPolicy}.
```

**Comment by paulinebessoles** [comment]
> Decidim-module-cookies is on crowdin so you must not add keys manually.

Thank you both for the answers! I added Dutch and Spanish in the Crowdin project.

File: config/locales/en.yml (hunk 1 above)
Comment (paulinebessoles) [reply config/locales/en.yml:14]: I checked with the v.0.21 (deployed on CultuurConnect) and it seems its `{privacyPolicy}`

Comment (armandfardeau) [reply config/locales/en.yml:14]: You're right, this is a js interpolation.

**Pull request merged**